dozer-sink-oracle/src/lib.rs

```diff
@@ -201,8 +200,10 @@ impl OracleSinkFactory {
                     FieldType::String | FieldType::Text,
                     OracleType::Varchar2(_) | OracleType::NVarchar2(_),
                 ) => {}
-                (FieldType::UInt | FieldType::Int, OracleType::Number(precision, 0))
+                (FieldType::U128 | FieldType::I128, OracleType::Number(precision, 0))
```

Comment:
This is incorrect. We are checking here that the output column is suitable for storing this data type. For i128/u128, that means we need precision to be at least 39. For UInt/Int, that means precision needs to be at least 20.

Reply:
Got it. Fixed condition.